README.md

```diff
@@ -45,7 +45,8 @@ class GoogleTagManager extends React.Component {
         const gtm = gtmParts({
             id: this.props.gtmId,
             dataLayerName: this.props.dataLayerName || 'dataLayer',
-            additionalEvents: this.props.additionalEvents || {}
+            additionalEvents: this.props.additionalEvents || {},
+            previewVariables: this.props.previewVariables || false
```

Comment:
Simply add `,` add at the end of the line like so :
`previewVariables: this.props.previewVariables || false,`.